packages/cli/README.md

```diff
@@ -218,6 +218,60 @@ Options:
   - `--ignore`: Ignores all of the fields on the list. Accepts all of the possible fields
   in the Web Manifest.
 
+## `fingerprint`
+
+Manages the list of fingerprints used to generate the Digital Asset Links file for the web application.
+
+Usage:
+
+```
+bubblewrap fingerprint [subcommand]
+``` 
+
+Global options:
+  - `--manifest=<manifest>`: path to the Trusted Web Activity configuration.',
+
+### Subcommands:
+
+#### `add`
+Adds a fingerprint to the project configuration.
+
+Usage:
+
+```
+bubblewrap fingerprint add [SHA-256 fingerprint] <flags>
+```
+
+Flags:
+ - `--name=<name>`: he name for the fingerprint.
+
+#### `remove`
+Removes a fingerprint from the project configuration.
+
+Usage:
+```
+bubblewrap fingerprint remove [SHA-256 fingerprint] <flags>
```

Comment:
For the above command you mention the flags below the example, but you don't do it here.

Reply:
There are no flags beyond the global for this command.

Reply:
Ah, OK, in which case, can you rename "Global Options" to "Global Flags" above, and rename Flags on line 245 to "Additional Flags"?